includes/Page_Template_Post_Type.php

```diff
@@ -50,8 +69,8 @@ class Page_Template_Post_Type extends Service_Base {
 	 * @return void
 	 */
 	public function register() {
-		$edit_posts   = $this->get_post_type_cap_name( Story_Post_Type::POST_TYPE_SLUG, 'edit_posts' );
-		$delete_posts = $this->get_post_type_cap_name( Story_Post_Type::POST_TYPE_SLUG, 'delete_posts' );
+		$edit_posts   = $this->get_post_type_cap_name( $this->story_post_type::POST_TYPE_SLUG, 'edit_posts' );
+		$delete_posts = $this->get_post_type_cap_name( $this->story_post_type::POST_TYPE_SLUG, 'delete_posts' );
```

Comment:
```suggestion
		$edit_posts   = $this->story_post_type->get_post_type_cap_name( $this->story_post_type::POST_TYPE_SLUG, 'edit_posts' );
		$delete_posts = $this->story_post_type->get_post_type_cap_name( $this->story_post_type::POST_TYPE_SLUG, 'delete_posts' );
```

Reply:
This does not work. `get_post_type_cap_name` is `protected`. If we wanna change this, then we can probably just get rid of that trait too. But in a separate PR.

Reply:
I was wonder about changing the Post Type trait an abstract base class for all post types. We can make post types a little more generic. When we can pass a   `story_post_type` object into classes that need to be capability checks.